src/Mjolnir.ts

```diff
@@ -479,6 +479,19 @@ export class Mjolnir {
         return await this.client.doRequest("POST", endpoint);
     }
 
+    public async suspendSynapseUser(userId: string): Promise<any> {
+        const endpoint = `/_synapse/admin/v1/suspend/${userId}`;
+        const body = {"suspend": true}
+        return await this.client.doRequest("PUT", endpoint, null, body);
+    }
+
+    public async unSuspendSynapseUser(userId: string): Promise<any> {
```

Comment:
```suggestion
    public async unsuspendSynapseUser(userId: string): Promise<any> {
```

very minor nit: typically we'd consider the reverse part of the same word

Reply:
(applies throughout)